IPs/RSKIP170.md

```diff
@@ -0,0 +1,91 @@
+# Peg-in to any address
+
+|RSKIP          |170           |
+| :------------ |:-------------|
+|**Title**      |Peg-in to any address |
+|**Created**    |01-SEP-20 |
+|**Author**     |MI |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Abstract
+
+This RSKIP proposes a way for users to be able to lock funds from BTC, indicating the RSK address in which they want their RBTCs transferred to and a BTC refund address in case the peg-in can not be completed for some reason.
+
+## Motivation
+
+Allowing users to lock funds to any address in RSK, EOA or contract.
+
+## Specification
+
+This RSKIP proposes an extension to the current pegin workflow. By including an output with an OP_RETURN op code and certain payload in the transaction sent to the federation to perform the peg-in, the user can determine the RSK address where the RBTCs will be transferred to. 
+
+The BTC transaction sent to the federation must contain 
+ one output with value 0 and OP_RETURN op code followed by the following data:
+- `RSKT` (for RSK  transfer) as prefix indicating the output is for RSK (4) bytes.
```

Comment:
I think the idea is to store the result of `bytes4(hash(RSKT))`. If that's correct please specify the expected value en hexadecimal